Submodels/MTSOL_call.py

```
@@ -170,52 +223,81 @@ def SetOperConditions(self,
+        self.StdinWrite(f"M {self.operating_conditions['Inlet_Mach']}")
 
         # Set critical amplification factor to N=9 rather than the default N=7
-        self.process.stdin.write(f"N {self.operating_conditions['N_crit']}\n")
-        self.process.stdin.flush()
+        self.StdinWrite(f"N {self.operating_conditions['N_crit']}")
 
         # Set the Reynolds number to 0 to ensure an inviscid solve is performed initially
-        self.process.stdin.write("R 0 \n")
-        self.process.stdin.flush()
-        
+        self.StdinWrite(f"R 0")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)